docs/source/tutorials/tabular.ipynb

```diff
@@ -456,6 +454,10 @@
    "source": [
     "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "\n",
+    "highlighted_indices = [827, 637]  # check these examples were top 5 in find_label_issues\n",
```

Comment:
Confirm these are the **only** 2 specific examples you ever reference / discuss / display in this notebook. If you mention other specific examples, their indices need to be in this list as well.

Reply:
Verified - it's just these 2!